.github/workflows/uncrustify.yml

```diff
@@ -7,66 +7,15 @@ on:
     paths:
       - '**.cpp'
       - '**.hpp'
-      - '.h'
-      - '.cxx'
+      - '**.h'
+      - '**.cxx'
 
 jobs:
   uncrustify:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v3
-        with:
-          path: src/Fast-DDS-QoS-Profiles-Manager
-
-      - uses: actions/checkout@v3
-        with:
-          repository: eProsima/eProsima-CI
-          path: src/eProsima-CI
-
-      - name: Fetch all branches and tags
-        run: |
-          cd src/Fast-DDS-QoS-Profiles-Manager
-          git fetch --prune --unshallow
-
-      - name: Install apt packages
-        uses: ./src/eProsima-CI/.github/actions/install-apt-packages
-
-      - name: Install Python packages
-        uses: ./src/eProsima-CI/.github/actions/install-python-packages
-
-      - name: Install uncrustify
-        run: |
-          sudo apt update && sudo apt -y install \
-            uncrustify
-
-      - name: Clone ament_lint
-        run: |
-          git clone https://github.com/ament/ament_lint.git src/ament_lint
-
-      - name: Build ament_uncrustify
-        run: |
-          colcon build --packages-up-to ament_uncrustify
-
-      - name: Fetch uncrustify config file
-        run: |
-          curl \
-            -l https://raw.githubusercontent.com/eProsima/cpp-style/master/uncrustify.cfg \
-            -o uncrustify.cfg
-
-      - name: Get difference
-        run: |
-          cd src/Fast-DDS-QoS-Profiles-Manager
-          echo "MODIFIED_FILES=$(git diff --name-only origin/${GITHUB_BASE_REF} origin/${GITHUB_HEAD_REF} | grep -e '\.h' -e '\.hpp' -e '\.cpp' | tr '\n' ' ')" >> $GITHUB_ENV
-
-      - name: Check style
-        run: |
-          source install/local_setup.bash
-          cd src/Fast-DDS-QoS-Profiles-Manager
-          ament_uncrustify \
-            -c ../../uncrustify.cfg \
-            --language CPP \
-            --xunit-file ../../uncrustify_results.xml \
-            ${MODIFIED_FILES}
+      - name: eProsima uncrustify
+        uses: eProsima/eProsima-CI/actions/uncrustify@main
 
       - name: Upload uncrustify results
```

Comment:
This upload should have at least a `path` argument. Also a name is pretty advisable.

Reply:
These arguments are already there.